### ISSUE TITLE
Create player manual

### ISSUE THREAD

**Issue by fr1tz** [fr1tz/terminal-overload#17]
TOL still has no player manual.

**Comment by mray** [comment]
Here is the current state of my simplified static player help for the website:

![help](https://cloud.githubusercontent.com/assets/164785/5828233/099fa1a4-a102-11e4-9c1d-2ae5107ebcd1.png)

**Comment by Evropi** [comment]
Very good! The colours and art are simply gorgeous.

Improvements to be made:
1. I think the discs screenshot should be changed so that the disc itself doesn't cover the player, and so there is a trail behind it. It is not so obvious in the current screenshot where the disc is
2. The second line of the 'Zone' section has a typo that makes the whole sentence really confusing. The second f**ro**m should be f**or**m.
3. I believe the entire HUD may need reworking. But that can be changed in a later iteration of the game. For now, you may want to make the HUD mockups as close to the game as possible.
4. I reckon you might want to change the icons for etherboarding (which should look more like a hoverboard) and super jumps (it looks like an explosion right now).
5. I'd remove the 'connected' part from the minimap. It is unnecessary and appears confusing. What about adding an `#eee` border between each zone instead?
6. I love the icons you have for 'Player form'. They are the best way to explain that game mechanic. But Etherform should be on top, as that is the form the players start in.

**Comment by mray** [comment]
1. The screenshots definitively need to be better, I just didn't find better screenshots in the demos I recorded. maybe animated gifs would be nice! - will have to remain on a to-do item for now.
2. Thanks - corrected.
3. That can better be discussed in [its separate issue](https://github.com/fr1tz/terminal-overload/issues)
4. Not sure what you mean. Maybe the two icons that explain Impulse? (at the very bottom) There is no icon just for etherboarding, just one for enhanced mobility.
5. Awesome remark - it can be removed.
6. They should indeed be switched.

great feedback!

**Comment by fr1tz** [comment]
I agree with Evropi and would simply add that some of the terminology is different from the manual and the actual game ("impulse" and "energy").

**Comment by Evropi** [comment]
To be honest, I kind of liked that terminology, but @fr1tz saw fit to switch it back to the old terms a few months ago, so nothing we can do about that I guess. :cry:

**Comment by fr1tz** [comment]
Let me put on my hat of pedantry: Neither ROTC nor TOL ever used the term "impulse" to refer to the mechanic that protects CATs against impulse (it was called "anchoring" in ROTC and "impulse damper" from the very start in TOL). And while ROTC used the term "Energy", the CAT module that reduced damage based on the amount of energy available was simply called "Damper". Since both mechanics work similarly (reducing impulse/damage resp.), TOL used "impulse damper"/"damage damper" from the very beginning and stuck with them (no "switching back" occured).

**Comment by mray** [comment]
Obviously I'm very interested in terminology, so may I suggest we move a discussion to a [more suitable issue](https://github.com/fr1tz/terminal-overload/issues/63)?

**Comment by fr1tz** [comment]
Sure

**Comment by mray** [comment]
I created a seperate issue #67
